search/process/tests.py

```diff
@@ -311,14 +311,14 @@ def test_acm_class(self):
         doc = transform.to_search_document(meta)
         self.assertEqual(doc.acm_class, ["F.4.1", "D.2.4"])
 
-    def test_metadata_id(self):
+    def test_doi(self):
```

Comment:
This test was getting skipped (and was incorrect), so I fixed it while I was at it. Unrelated to the bug.